catalogue/webapp/pages/search/works.tsx

```diff
@@ -123,23 +125,25 @@ export const CatalogueSearchPage: NextPageWithLayout<Props> = ({
       </Head>
 
       <div className="container">
-        <SearchFilters
-          query={queryString}
-          linkResolver={linkResolver}
-          searchFormId="searchPageForm"
-          changeHandler={() => {
-            const form = document.getElementById('searchPageForm');
-            form &&
-              form.dispatchEvent(
-                new window.Event('submit', {
-                  cancelable: true,
-                  bubbles: true,
-                })
-              );
-          }}
-          filters={filters}
-          isNewStyle
-        />
+        {isEnhanced && (
```

Comment:
Same as above for `inEnhanced` and the div

Reply:
solved